index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous" />
+   <link rel="stylesheet" type="text/css" href="./css/style.css" />
+   <title>Apple</title>
+</head>
+
+<body>
+   <div class="container">
+      <div class="content">
+         <nav class="toolbar">
+            <ul>
+               <li class="item"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/">
+                     <div class="apple_logo"></div>
+                  </a></li>
+               <li class="item"><a href="#">Store</a></li>
+               <li class="item"><a href="#">Mac</a></li>
+               <li class="item"><a href="#">iPod</a></li>
+               <li class="item"><a href="#">iPhone</a></li>
+               <li class="item"><a href="#">iPad</a></li>
+               <li class="item"><a href="#">iTune</a></li>
+               <li class="item"><a href="#">Support</a></li>
+               <li class="item">
+                  <div class="search"><i class="fa fa-search"></i><input type="text"></div>
+               </li>
+            </ul>
+         </nav>
+         <img
+            src="https://web.archive.org/web/20140301004336im_/http://images.apple.com/home/images/your_verse_title_small.png"
+            alt="image">
+
+         <img
+            src="https://web.archive.org/web/20140301004340im_/http://images.apple.com/home/images/your_verse_subhead_small.png"
+            alt="image">
+
+         <img
+            src="https://web.archive.org/web/20140301004341im_/http://images.apple.com/home/images/cta_your_verse_27pt.png"
+            alt="image">
```

Comment:
The same thing here as well, **USE TEXT** not a screenshot of the text
<img width="581" alt="Screen Shot 2020-09-28 at 6 29 59 PM" src="https://user-images.githubusercontent.com/5264771/94466065-be1b4300-01b8-11eb-9e5b-5099353e03a8.png">